app/scripts/metamask-controller.js

```diff
@@ -7721,6 +7723,12 @@ export default class MetamaskController extends EventEmitter {
           .showConfirmationAdvancedDetails;
       },
       getHDEntropyIndex: this.getHDEntropyIndex.bind(this),
+      getRemoteFeatureFlags: () => {
```

Comment:
If the reason is to access the state from UI, you can safely remove this function

Reply:
I will revisit this. TY

Reply:
Can you create an ticket to track this? As this is not applying to current usage of remote feature flags and I want to make sure it's removed to avoid confusion for other developers to user remote feature flags